testing/vcs/test_vcs_autot_axis_titles.py

```diff
@@ -18,8 +18,8 @@
 plot = sys.argv[3]
 x_over_y = sys.argv[4]
 if (x_over_y == '0.5'):
-    xSize = 400
-    ySize = 800
+    xSize = 250
+    ySize = 500
 else:
     xSize = 800
     ySize = 400
```

Comment:
The problem way the Y size was bigger than the screen size. The way it is now works fine on the lowest resolution on my mac: 1024 x 640